zerberus/zone.cpp

```diff
@@ -62,12 +62,13 @@ bool Zone::match(Channel* c, int k, int v, Trigger et, double rand)
                               return false;
                         }
                   }
-            if (et == Trigger::ATTACK) {
-                  ++seq;
-                  if (seq > seqLen)
-                        seq = 0;
-                  }
-            qDebug("seq %d seqPos %d seqLen %d", seq, seqPos, seqLen);
+
+
+            ++seq;
```

Comment:
you might be right... seq should probably just being increment if the cc/ccVal matches